pkg/asyncexec/stream.go

```diff
@@ -0,0 +1,60 @@
+package asyncexec
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+// StreamCommand executes a given command in a context directory and streams
+// the output to the according stdin/stdout.
+func StreamCommand(workDir string, command string, args []string) error {
+	cmd := exec.Command(command, args...)
+	if workDir != "" {
+		cmd.Dir = workDir
+	}
+
+	acmd := New(cmd, 256)
```

Comment:
Any reason behind the 256 buffer?

Reply:
Not really, just leaving space for commands sending message super fast..  is it too big? any suggestion for a better number?

Reply:
Seems fairly arbitrary to me. I don't have much of a point of reference for what kinds of commands we would be sending through here.

Reply:
The higher the number the more chance we have to print the stream of output in the right order.. thinking fast alternate streams of stderr/stdout. But still yes we can reduce the number, 8/16/32 should be fair.